packages/react-ui/src/app/routes/flows/index.tsx

```diff
@@ -324,7 +380,7 @@ const FlowsPage = () => {
                     hasPermission={userHasPermissionToPushToGit}
                   >
                     <PushToGitDialog
-                      flowIds={selectedRows.map((flow) => flow.id)}
+                      flowIds={selectedRowIds.map((flow) => flow.version.id)}
```

Comment:
rename selectedRowIds to selectedRows because that's what they are, if you want them to be selectedRowIds, change the type to string array and set the value to the ids only in the setter, and do this in all the components that have that variable name.